docs/book/src/clusteropenstack/configuration.md

```diff
@@ -243,12 +243,20 @@ metadata:
   namespace: <cluster-name>
```

Comment:
L182: `please note that it is possible to get multiple networks as a result.` But network filter must only return one network in my understanding.
YAML example should change in Network Filters, Multiple Networks, and Subnet Filters.

Reply:
the source of confusion you are having is that there is a difference between the networking api and the ports api right now. We intend to remove networks all together and pivot all the way to just using ports. That is going to be handled in a separate issue, which is why I chose to leave this alone for now

Reply:
Network and subnet queries for ports should only return one value. But for networks and subnets, we have decided to implement this weird poweruser api.

Reply:
for more on what is to come, see https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/1023

Reply:
Yes, I see #1023 and asked. I understand I missed this is half of fixing #1023, thanks.